src/utils/trust/common.trust.data.mapper.ts

```diff
@@ -1,12 +1,19 @@
 import { ApplicationData } from '../../model';
 import { CommonTrustData } from '../../model/trust.page.model';
 import { getTrustByIdFromApp } from '../trusts';
+import { getReviewTrustById } from '../../utils/update/review_trusts';
 
 const mapCommonTrustDataToPage = (
   appData: ApplicationData,
   trustId: string,
+  isReview: boolean
 ): CommonTrustData => {
-  const trustData = getTrustByIdFromApp(appData, trustId);
+  let trustData;
```

Comment:
const trustData = isReview ? getReviewTrustById(appData, trustId) : getTrustByIdFromApp(appData, trustId);

Reply:
Changing to the suggested code produces errors in the return statement, will just keep to what is here to prevent any further changes